pom.xml

```diff
@@ -368,16 +373,33 @@
 						</goals>
 					</execution>
 					<execution>
-						<id>default-report</id>
+						<id>merge-data-files</id>
+						<phase>post-integration-test</phase>
 						<goals>
-							<goal>report</goal>
+							<goal>merge</goal>
 						</goals>
+						<configuration>
+							<fileSets>
+								<fileSet>
+									<directory>${project.build.directory}</directory>
+									<includes>
+										<include>jacoco.exec</include>
+										<include>jacoco-it.exec</include>
+									</includes>
+								</fileSet>
+							</fileSets>
+							<destFile>${project.build.directory}/jacoco-merged.exec</destFile>
+						</configuration>
 					</execution>
 					<execution>
-						<id>default-report-integration</id>
+						<id>merged-report</id>
+						<phase>post-integration-test</phase>
 						<goals>
-							<goal>report-integration</goal>
+							<goal>report</goal>
 						</goals>
+						<configuration>
+							<dataFile>${project.build.directory}/jacoco-merged.exec</dataFile>
+						</configuration>
 					</execution>
 				</executions>
 			</plugin>
```

Comment:
Do these lines use tabs instead of spaces?

They look different here:

<img width="635" alt="image" src="https://user-images.githubusercontent.com/91831478/196398616-3f070701-e263-4b79-adb6-d91ae903522d.png">

Reply:
yes it seems to be an error. Fixing this.